### DISCUSSION TITLE
does cx_freeze make it hard to reverse engineer the code

### DISCUSSION THREAD

**Discussion by kthehatter** [marcelotduarte/cx_Freeze#1268] [Q&A]
just as the title says i was wondering i cx_freeze has some degree of protection for my python code and is it possible to do obsuce operation on it to make it even harder to decompile

**Comment by marcelotduarte** [comment]
See #935

**Reply by kthehatter** [reply]
hello i tried using pyarmor to Obfuscation  the code it worked just fine but then when I used cx_freeze with the Obfuscated code it gives an error
![image](https://user-images.githubusercontent.com/13066001/138008754-32306f59-34a4-4c79-a8ef-c6cacb22bbf1.png)

**Reply by marcelotduarte** [reply]
Interesting. Please, make a test with [samples](https://github.com/marcelotduarte/cx_Freeze/tree/main/cx_Freeze/samples)
Modify simple sample or sqllite sample to test and create a [new issue](https://github.com/marcelotduarte/cx_Freeze/issues/new/choose). I'll check it out when I have time.

**Reply by Stefan0633** [reply]
Hi,mate. I also run into the same problem. I wonder if you have fixed the issue. Appreciate if you can help me.

**Reply by kthehatter** [reply]
hello @Stefan0633 i wasn't able to fix it and moved away from trying to Obfuscate the code
